src/components/enter-email/enter-email-controller.ts

```diff
@@ -168,6 +170,18 @@ export function enterEmailCreatePost(
     );
 
     if (!sendNotificationResponse.success) {
+      if (
```

Comment:
I don't think I understand why we need this here - doesn't `getErrorPathByCode` below do the same thing? Could we need read `isAccountCreationJourney` from the session in the security code controller?

Reply:
(Same applies to other similar changes)